cmd/notary-server/config.json

```diff
@@ -3,10 +3,7 @@
 		"addr": ":4443"
 	},
 	"trust_service": {
-		"type": "remote",
```

Comment:
Actually I would like to discuss this. I think the master branch of notary should work with dev out of the box. You docker-compose up and run notary.

Production is a different beast, and we shouldn't be surprised if we have to change the configs for it to work there.